README.md

```diff
@@ -37,7 +37,14 @@ Read more about jsDeliver versioning on their [website](http://www.jsdelivr.com/
 
 ## Configuration
 
-No adapter configuration is currently available, read the [Chart.js documention](https://www.chartjs.org/docs/latest) for other possible date/time related options. For example, the time scale [`time.*` options](https://www.chartjs.org/docs/latest/axes/cartesian/time.html#configuration-options) can be overridden using the [date-fns tokens](https://date-fns.org/v1.30.1/docs/format).
+Locale support:
```

Comment:
Markdown will not break to the next line after `support:`. 

I would create a sub-section instead for better anchoring:

```
### Locale support:

date-fns, contrary to moment.js ...
```